src/script/calling/CallingRepository.ts

```diff
@@ -574,27 +586,26 @@ export class CallingRepository {
     } = event;
     const isFederated = this.core.backendFeatures.isFederated && qualified_conversation && qualified_from;
     const userId = isFederated ? qualified_from : {domain: '', id: from};
-    const conversationId = isFederated ? qualified_conversation : {domain: '', id: conversation};
     const currentTimestamp = this.serverTimeHandler.toServerTimestamp();
     const toSecond = (timestamp: number) => Math.floor(timestamp / 1000);
     const contentStr = JSON.stringify(content);
-    const conversationEntity = this.conversationState.findConversation(conversationId);
+    const conversationId = this.extractTargetedConversationId(event);
+    const conversation = this.conversationState.findConversation(conversationId);
 
-    if (!conversationEntity) {
+    if (!conversation) {
```

Comment:
Conversation can never be undefined?

Reply:
Well it can if the ID in the event payload doesn't correspond to a conversation we have locally